app/jobs/stream_versions_to_big_query_job.rb

```diff
@@ -1,9 +1,13 @@
 class StreamVersionsToBigQueryJob < ApplicationJob
+  delegate :dfe_analytics_enabled?, to: Feature
+
   self.log_arguments = false
 
   queue_as :low_priority
 
   def perform(user_name, data)
+    return unless dfe_analytics_enabled?
```

Comment:
I've done this check within the Job but I can see an argument for doing it in the module and never creating the job? Thoughts